backup_to_s3.py

```diff
@@ -100,13 +100,13 @@ def save_url_image_to_path(self, state, data_url, path):
         }
 
         # PhantomJScloud gets the page length wrong for some states, need to set those manually
-        if state in ['ID', 'PA', 'IN', 'CA']:
+        if state in ['ID', 'PA', 'CA']:
             logger.info(f"using larger viewport for state {state}")
             data['renderSettings'] = {'viewport': {'width': 1400, 'height': 3000}}
-        elif state in ['NE']:
+        elif state in ['IN','NE']:
             # really huge viewport for some reason
             logger.info(f"using huge viewport for state {state}")
-            data['renderSettings'] = {'viewport': {'width': 1400, 'height': 5000}}
+            data['renderSettings'] = {'viewport': {'width': 1400, 'height': 7000}}
```

Comment:
@muamichali thanks for this! It ended up taking 8500 to capture the whole thing (annoyingly) so I made a yet another special case bit for this and merged it